src/components/Navbar.tsx

```diff
@@ -1,4 +1,14 @@
+import * as RRD from 'react-router-dom';
+import cn from 'classnames';
+
+const getLinkClass = ({ isActive }: { isActive: boolean }) => (cn(
+  'navbar-item', { 'has-background-grey-lighter': isActive },
+));
+
 export const Navbar = () => {
+  const [searchParams] = RRD.useSearchParams();
+  const { state } = RRD.useLocation();
```

Comment:
```suggestion
  const { state } = useLocation();
```

FYI you can destructure methods and components from lib so you don't need this RRD.